appveyor.yml

```diff
@@ -1,5 +1,5 @@
 init:
-  - git config --global core.autocrlf true
+  - git config --global core.autocrlf input
```

Comment:
Yup, we use TeamCity. AppVeyor builds are just for PR verification.

We don't control the `git clone` command directly. Instead, TeamCity has a "Convert line-endings to CRLF" which is _not_ enabled, meaning the system should work like `core.autocrlf=false`. Not sure why tests with poor line-ending expectations aren't failing.